### PR TITLE
updated logic on reading ascii waveforms

### PR DESCRIPTION
**Pull request by shot0829**

### CODE REVIEW

**Review by jpa198 — COMMENTED** [review]
Wasn't there also a change from floor to round?

**Comment by shot0829** [comment]
> Wasn't there also a change from floor to round?

since we are now forcing DT to be float32, the round change is not required.

File: IM_calculation/IM/read_waveform.py [hunk 1]
```diff
@@ -70,7 +77,7 @@ def read_ascii_header(fid):
     # second line of the header
     header2 = next(fid).split()
     NT = np.int(header2[0])
-    DT = np.float(header2[1])
+    DT = np.float32(header2[1])
```
Comment (sungeunbae) [inline IM_calculation/IM/read_waveform.py:80]: Add a comment here for future reference

**Review by sungeunbae — DISMISSED** [review]
Add some comments

**Review by jasonmotha — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Pull request merged**